Assets/Plugins/CountlySDK/Services/CrashReportsCountlyService.cs

```diff
@@ -17,6 +20,7 @@ internal CrashReportsCountlyService(CountlyConfiguration configModel, RequestCou
         {
             _configModel = configModel;
             _requestCountlyHelper = requestCountlyHelper;
+            _startTime = DateTimeOffset.UtcNow.ToUnixTimeSeconds();

```

Comment:
set this as a static field on the lines that declares this variable

Reply:
Only one instance of CrashReportsCountlyService is created in our sdk. Do we need to make it static.

Reply:
You still didn't resolve this. Move the setting of this value to it's decleration.